backend/main.py

```diff
@@ -3,6 +3,7 @@
 from starlette.endpoints import HTTPEndpoint
 from starlette.requests import Request
 from starlette.responses import JSONResponse
+from starlette.responses import RedirectResponse
```

Comment:
fix imports

Reply:
from star.responses import Foo, Bar

not from star.responses import Foo
and from star.res import Bar